tests/Feature/Events/APIv2EventTest.php

```diff
@@ -68,21 +74,30 @@ public function testGetEventsForGroup(): void {
         $this->assertTrue(array_key_exists('stats', $json['data']));
         $this->assertTrue(array_key_exists('updated_at', $json['data']));
 
-        // Check unapproved.  Past events show for moderation, earliest first.
-        $response = $this->get("/api/v2/moderate/events?api_token=1234");
+        // Check unapproved events via the moderation endpoint
+        $response = $this->get("/api/v2/moderate/events?api_token=$user->api_token");
         $response->assertSuccessful();
-        $json = json_decode($response->getContent(), true);
-        self::assertEquals(2, count($json));
-        $this->assertEquals($id2, $json[0]['id']);
-        $this->assertEquals($id1, $json[1]['id']);
-        self::assertFalse($json[0]['approved']);
-        self::assertFalse($json[1]['approved']);
+        $moderationEvents = json_decode($response->getContent(), true);
+        
+        // Extract all event IDs from the response
+        $eventIds = collect($moderationEvents)->pluck('id')->toArray();
+        
+        // Verify our created events are in the moderation list
+        $this->assertContains($id1, $eventIds, "First created event should be in the moderation list");
+        $this->assertContains($id2, $eventIds, "Second created event should be in the moderation list");
+        
+        // Verify our events aren't approved
+        $ourEvents = collect($moderationEvents)->filter(function($event) use ($id1, $id2) {
```

Comment:
I'm not familiar with the `collect` method.

Reply:
collect would be similar to the early days of lodash and JS arrays, so it wraps an "array" and provides useful helper methods to interact with it.

Reply:
Okay 👍 

I think this could be
```
$ourEvents = array_filter(...);
```

And above, `collect($moderationEvents)->pluck('id')->toArray();` could be:

```
$eventIds = array_map(($event) => $event['id'], $moderationEvents);
```

Reply:
Addressed in [28c7c8c](https://github.com/iFixit/restarters/pull/7/commits/28c7c8c3c7037f6ce6c18a489dbb8963fd99241f)